.gitignore

```diff
@@ -33,6 +33,9 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+# Ignore credentials.yml.enc
+/config/credentials.yml.enc
```

Comment:
Even though you have ignored it I can still see this as part of PR, can you fix that
https://github.com/rt4914/KlinicCon-Backend/pull/62/files

Reply:
Hi, here got to know that, I can't ignore it if it is already present in main branch, so I have kept it now as it is & also removed the` .gitignore `command.
Below is the details reason: 
`if your config/credentials.yml.enc file is still being pushed to the repository after you've added it to .gitignore, it's likely because Git is already tracking the file. Files that have already been committed to Git won't be ignored by simply adding them to .gitignore. You need to remove the file from the Git index while keeping it in your local file system.`